web/app_behat.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Request;
+
+$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
```

Comment:
at least the require_once is.

For the loader, it can be used for the ApcLoader